test/integration/targets/postgresql/tasks/postgresql_privs.yml

```diff
@@ -397,10 +398,12 @@
     login_password: password
   register: result
   ignore_errors: yes
+  when: result.failed == false
```

Comment:
Why? Seems it should be removed. Otherwise if the tests failed the assertions will be skipped

Reply:
Needs find and solve problem

Reply:
The Create table statement will throw an error for PG versions < 10

Reply:
1. remove when: result.failed
2. use when: postgres_version_resp.stdout is version('10', '>=') instead

Reply:
just change it

Reply:
and all will be OK :) I hope

Reply:
find info about embedded partitionting is available from PG 10
and add a note about it in comment to your tests.

Reply:
That's the better solution ;-)

Reply:
welcome